docs/docs/glossary/progressive-enhancement.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Progressive Enhancement
+disableTableOfContents: true
+---
+
+Learn what _progressive enhancement_ is and how Gatsby it easy to build sites using progressive enhancement principles.
+
+## What is progressive enhancement?
+
+_Progressive enhancement_ is a strategy for building web sites in which core functionality is available to all browsers, while non-critical enhancements are available to capable browsers. For example, a progressively-enhanced form submission might trigger a `fetch` network request in browsers that support the Fetch API, and a traditional form submission with a full page reload in browsers that do not.
+
+Progressive enhancement can also ensure that your site is usable even if JavaScript fails to load. Poor network conditions, firewalls, and some browser settings can prevent JavaScript from executing. If your site relies entirely on JavaScript and client-side rendering, visitors may see a blank screen while waiting for JavaScript to load.
+
+You'll sometimes see progressive enhancement discussed alongside _graceful degradation_. Sites built to degrade gracefully are optimized for the latest browsers, but ensure that core functions are still available to older browsers.
+
+### How Gatsby enables progressive enhancement
+
+Gatsby uses [server-side rendering](/docs/glossary/server-side-rendering/) and a React feature known as [hydration](/docs/glossary/hydration/) to create progressively-enhanced sites. During the build process, Gatsby creates an HTML document for each URL of your site and a corresponding React component.
```

Comment:
Would you mind taking another look at the last sentence here? I'm not sure exactly what it should be but "and a corresponding React component" feels like it's not quite in the right place.